app/models/payment.rb

```diff
@@ -1,2 +1,3 @@
 class Payment < ApplicationRecord
+    enum category:{food:0,daily:1,transport:2,entertainment:3,education:4,insurance:5,medical:6,communication:7,utilities:8,rent:9,saving:10,other:11,salary:12}
```

Comment:
とても良いのですが、少し横長なので下記のように改行して見やすくしましょう
```suggestion
    enum category: {   
        food: 0,
        daily: 1,
        transport: 2,
    }
```

Reply:
流石に見えづらいなとは思ってました。
改行が使えるのですね、修正します。